src/marqo/index.py

```diff
@@ -404,6 +404,7 @@ def embed(self, content: Union[Union[str, Dict[str, float]], List[Union[str, Dic
         )
         body = {
             "content": content,
+            "content_type": content_type,
```

Comment:
Do you still want to include this in the body even if it is `None`?

Reply:
Yeah I think it's alright. the embed endpoint in Marqo handles `None`, `query`, or `document`

Reply:
I am not 100% sure about this point. I would say we should omit it because we are omitting `image_download_headers` and `model_auth`.  @farshidz Can you confirm this?

Reply:
I made its inclusion conditional like `image_download_headers` and `model_auth`

Reply:
Actually, we really do need to include content_type even if it is None, according to the expected cases from Joshua and Owen